src/index.ts

```diff
@@ -60,11 +58,11 @@ async function run() {
 
     const persona = db.persona.get(newPotd!.personaId)
     log.info(`New POTD is ${db.persona.get(newPotd!.personaId)?.title}`)
-    potd = persona!
+    forcePotdChange(persona!)
   }, null, false, 'Europe/Paris')
   potdJob.start()
 
-  client.on(Events.MessageCreate, async (message: Message) => handlePrompt(client, message, potd))
+  client.on(Events.MessageCreate, async (message: Message) => handlePrompt(client, message, loadPotd({ forcePersonaId: args.forcePersona })))
```

Comment:
As the potd can change anytime, we have to check it on every prompt, maybe there is a cleaner way ?